source/routing/raptor.cpp

```diff
@@ -339,8 +335,8 @@ Journey convert_to_bound(const StartingPointSndPhase& sp, bool clockwise) {
 }  // namespace
 
 // copy and do the off by one for strict comparison for the second pass.
-static IdxMap<type::StopPoint, DateTime> snd_pass_best_labels(const bool clockwise,
-                                                              IdxMap<type::StopPoint, DateTime> best_labels) {
+static IdxMap<type::StopPoint, DateTime>& snd_pass_best_labels(const bool clockwise,
```

Comment:
Before this function was copying its input, modifying the copy, and returning the copy.
Now it modify its input and returns it.
Is it ok ?

Reply:
It's now modifying the copy provided by ` const auto inrows_labels = best_labels.inrow_labels();` (cf you comment below :) )